### PR TITLE
Solved String1.py

### PR DESCRIPTION
**Pull request by Valenciahub**

### CODE REVIEW

File: google-python-exercises/basic/string1.py [hunk 1]
```diff
@@ -24,89 +24,90 @@
...
 # Assume a and b are length 2 or more.
 def mix_up(a, b):
-  # +++your code here+++
-  return
+    # +++your code here+++
+    return '0'
 
 
-# Provided simple test() function used in main() to print
+# Provided sime test() function used in main() to print
```
Comment (dancape) [inline google-python-exercises/basic/string1.py]: Not a big deal but spelling is wrong

Comment (MichaelPichardo) [reply google-python-exercises/basic/string1.py]: yeah

**Comment by Valenciahub** [comment]
I have fixed the mistake

File: google-python-exercises/basic/string1.py [hunk 2]
```diff
@@ -85,22 +85,22 @@ def main():
     test(donuts(10), 'Number of donuts: many')
     test(donuts(99), 'Number of donuts: many')
 
-    print()
+    print
```
Comment (dancape) [inline google-python-exercises/basic/string1.py]: This is wrong it was correct before

**Pull request merged**